clients/client-python/pylintrc

```diff
@@ -53,3 +50,17 @@ logging-modules=logging
 # system, and so shouldn't trigger E1101 when accessed. Python regular
 # expressions are accepted.
 generated-members=gravitino.utils.http_client.Response
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=120
```

Comment:
II would suggest to change to 100 to align with our java code. WDYT @noidname01 @xunliu , what is the typical length of python code?

Reply:
This is because `Black` seems not to format the length of docstrings but `Pylint` will complain about it.
There are many long docstrings in current project, I can try to refine them, But I think this PR has done too many modifications, maybe finish in follow-up PRs ?

BTW, the max length is defined as **79** characters each line in PEP8(Python Standard Library Code Style), and the default setting of Black is **88**.

Reply:
Yeah, you can do it in a separate PR. the default length seems too short, if so I would suggest to align with Java code to use 100.